### PR TITLE
Add upload-options for lokalise-file-push

### PR DESCRIPTION
**Pull request by igorDolzh**
https://linear.app/pleo/issue/INT-41/update-enjson-in-pleo-mobile-doesnt-come-to-localisecom

It will help to pass any options for file uploading including an ability to track updated strings

### CODE REVIEW

File: lokalise-file-push/lib/main.js [hunk 1]
```diff
@@ -58,14 +59,8 @@ function uploadFiles({ lokalise, projectId, filePath, tags, locales, callback, }
...
                 const buff = Buffer.from(file);
-                let process = yield lokalise.files.upload(projectId, {
-                    data: buff.toString("base64"),
-                    filename,
-                    lang_iso: lang,
-                    tags,
-                    //@ts-ignore
-                    convert_placeholders: false,
-                });
+                const options = uploadOptions ? JSON.parse(uploadOptions) : {};
```
Comment (bel0v) [inline lokalise-file-push/lib/main.js:62]: JSON.parse must be wrapped in try catch

File: lokalise-file-push/src/main.ts [hunk 2]
```diff
@@ -54,13 +55,14 @@ async function uploadFiles({
       const filename = filePath.replace(LANG_ISO_PLACEHOLDER, lang);
       const file: Blob = await readLanguageFile(filename);
       const buff = Buffer.from(file);
+      const options = uploadOptions ? JSON.parse(uploadOptions) : {}
```
Comment (bel0v) [inline lokalise-file-push/src/main.ts:58]: same here

**Review by bel0v — APPROVED** [review]
couple of comments, otherwise cool 👍

**Review by Wiingaard — APPROVED** [review]
LGTM 👍 Thanks for doing this man 💪

**Pull request merged**